Classes/Event/EventMetadata.php

```diff
@@ -1,112 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-/**
- * The EventMetadata is a container for arbitrary metadata for Events.
- * This class is immutable.
- *
- * @api
- */
-class EventMetadata
```

Comment:
I don't see it as a "thingy" to have immutable event (we can't guarantee that, but should lead by example) and metadata. Conceptually an Event is the the definition of immutable data, and even though technically it might be mutated during it's creation process, the data structure shouldn't be tailored for this, but rather for the 99% of it's lifetime *after* being stored, where it is indeed immutable.